tests/flow/test_comprehension_functions.py

```diff
@@ -1,11 +1,20 @@
-import re
 import redis
 from RLTest import Env
 from base import FlowTestsBase
 from redisgraph import Graph, Node, Edge
+from redisgraph.execution_plan import ExecutionPlan
+from execution_plan_util import locate_operation
 
 redis_graph = None
 
+def _check_pattern_comprehension_plan(plan: ExecutionPlan):
+    apply = locate_operation(plan.structured_plan, "Apply")
```

Comment:
I was thinking we're going to construct a subtree, using the Operation class
```python
op.append_child(Operation(...
```
Once we have the pattern we pass it into a locate pattern together with the execution-plan